### PR TITLE
[WOR-1461] Emit events for clipboard copy events (WOR-1461).

### PR DESCRIPTION
**Pull request by cahrens**
JIRA ticket: https://broadworkbench.atlassian.net/browse/WOR-1461

Adds events for clipboard copy events on the dashboard, as we wish to see how often people click them.

Azure:
image

GCP:
image

### CODE REVIEW

File: src/workspaces/dashboard/AzureStorageDetails.ts [hunk 1]
```diff
@@ -37,6 +40,11 @@ export const AzureStorageDetails = (props: AzureStorageDetailsProps): ReactNode
         'aria-label': 'Copy resource group ID to clipboard',
         text: azureContext.managedResourceGroupId,
         style: { marginLeft: '0.25rem' },
+        onClick: (_) => {
```
Comment (cahrens) [inline src/workspaces/dashboard/AzureStorageDetails.ts:43]: Note that `onClick` doesn't prevent the default action (copying to clipboard) from happening.

**Comment by cahrens** [comment]
The IA integration tests are failing in dev/staging as well, and I have notified the IA team.

**Review by blakery — APPROVED** [review]

File: src/workspaces/dashboard/AzureStorageDetails.test.ts [hunk 2]
```diff
@@ -1,11 +1,32 @@
 import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import * as clipboard from 'clipboard-polyfill/text';
```
Comment (sergiygetlin) [inline src/workspaces/dashboard/AzureStorageDetails.test.ts:3]: is this a normal pattern to use * for import?

**Review by sergiygetlin — APPROVED** [review]
LGTM; just question about * import.

File: src/workspaces/dashboard/AzureStorageDetails.test.ts (hunk 2 above)
Comment (cahrens) [reply src/workspaces/dashboard/AzureStorageDetails.test.ts:3]: No, not in our "regular" code (though it sometimes still is done for utils files). But it is necessary here for mocking.

**Pull request merged**